airflow/task/task_runner/standard_task_runner.py

```diff
@@ -53,6 +55,11 @@ def start(self):
         else:
             self.process = self._start_by_exec()
 
+        if self.process:
+            log_reader = threading.Thread(target=self._read_task_utilization)
```

Comment:
`log_reader` is a strange name for this, it's not really reading any logs?

maybe `resource_monitor`?

Reply:
Good catch! Copy pasta :|